recipes/castorapi/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "castorapi" %}
+{% set version = "0.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 4544ab11e1144c1a66a466f5ffc379ff1031babe6d74c0fba027f3d4fd6d3b0c
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
```

Comment:
In general, `pip` and `python` are the only dependencies that need to be in the `host` sections, while the remaining ones go to `run`.

Reply:
I just saw that you are using:
```
  install_requires=[            # I get to this in a second
            'io',
            'json',
            'os',
            'pandas',
            'requests',
            'progressbar',
            'logging'
      ],
```
in `setup.py`. In that case, you can leave the remaining dependencies here.

Reply:
(changed nothing in the requirements now; thought that's what you meant)